optaplanner-examples/pom.xml

```diff
@@ -73,6 +73,36 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin> <!-- Shade plugin configuration shared by all distribution modules. -->
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.2</version>
```

Comment:
Why are we adding the shade plugin?
I'd like to avoid the shade plugin at all cost.